bentoml/artifact/tf_keras_model_artifact.py

```diff
@@ -68,3 +69,30 @@ def save(self, dst):
 
     def get(self):
         return self._model
+
+
+class _TfKerasModelWrapper(object):
+
+    def __init__(self, model):
+        try:
+            import tensorflow as tf
+            from tensorflow.python.keras.models import load_model
+        except ImportError:
+            raise ImportError("tensorflow package is required to use TfKerasModelArtifact")
+        self.session = tf.Session()
+        self.graph = tf.get_default_graph()
+        with self.graph.as_default():
+            with self.session.as_default():
+                self.model = load_model(model)
+
+    def predict(self, x):
+        with self.graph.as_default():
+            with self.session.as_default():
+                result = self.model.predict(x)
+        return result
+
+    def predict_classes(self, x):
```

Comment:
this is not a `training.Model` method, only `Sequential` has it

Reply:
also make sure to pass down all parameters for this type of wrapper  `def predict_classes(self, x, batch_size=32, verbose=0):`

Reply:
I think might be better to force gloabl default graph for entire API function instead of wrapping individual keras model methods here

Reply:
I was afraid of using it as global default graph, but I will test that route.

Reply:
Another pro of wrapping the individual keras model is there is no additional actions from user.  If we use global graph, user have to make prediction call within that graph context.  I think it would be just strange to users to use a var that 'comes out' from nowhere.

```python
...
def predict(self, df):
      with graph.as_default():    # user has no knowledge of where this come from.
              result = self.artifacts.model.predict(df)
```

Reply:
@yubozhao oh I didn't mean "global" graph, I was referring to self.graph here. But instead of wrapping on Keras model, wrap it around the API function.

Reply:
Btw, I have some new changes on how user API function access artifacts, that might make this implementation easier.

Reply:
Oh. I got you.  Also. I totally forgot to mention to you that this PR is not mean to be merged. I was able to get the tf keras model working with this, but I don’t think this is a good solution and there are going to be potential problems down the line. This PR’s goal is start to have a conversation around this